lib/rubocop/cop/root_cops/envvar_assignment.rb

```diff
@@ -0,0 +1,37 @@
+module RuboCop
+  module Cop
+    module RootCops
+      class EnvvarAssignment < Cop
+        # Assigning an ENVVAR to a constant has unexpected behavior when used with set_environment_variable
```

Comment:
Consider including this in the message

Reply:
Just that first sentence you mean?

Reply:
I meant the entire comment, but don't feel like you have to. I was debating whether that context would be helpful to have when the error pops up 🤔

Reply:
Thoughts?

Reply:
It's definitely helpful context. I'm just concerned about the message being too long, it seems like all of the messages I've seen tend to just be 1-2 sentences. I don't have any better reason than that though.

Reply:
That makes sense. I'm fine leaving it as-is then 👍

Reply:
I added the first sentence to the message since I think some more explanation was definitely needed.